client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -104,8 +107,5 @@ export default connect(
 	( state ) => ( {
 		userId: getTwoFactorUserId( state ),
 		twoStepNonce: getTwoFactorAuthNonce( state ),
-	} ),
-	{
-		loginUserWithTwoFactorVerificationCode,
```

Comment:
`loginUserWithTwoFactorVerificationCode` does not dispatch any actions. We will update it such that it does when we add error handling to this flow, but there was an error in our middleware when this was dispatched (presumably because it was not a valid thunk), so for now, we can just call this directly.